Client/Frontend/Home/PrivateHome/PrivateHomepageViewController.swift

```diff
@@ -0,0 +1,131 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Common
+import UIKit
+import Shared
+
+// Displays the view for the private homepage when users create a new tab in private browsing
+final class PrivateHomepageViewController: UIViewController, ContentContainable, Themeable {
+    enum UX {
+        static let scrollContainerStackSpacing: CGFloat = 24
+        static let defaultScrollContainerPadding: CGFloat = 16
+        private static let iPadScrollContainerPadding: CGFloat = 164
+
+        static var scrollContainerPadding: CGFloat {
+            let isiPad = UIDevice.current.userInterfaceIdiom == .pad
+            return isiPad ? UX.iPadScrollContainerPadding : UX.defaultScrollContainerPadding
```

Comment:
Should this be different on iPad in multitasking?

Reply:
good point! I'm still working on some padding clarifications with Juhee, and will bring this up as well. Any updates to padding, I'll make in a new PR.

Ref: https://github.com/mozilla-mobile/firefox-ios/pull/17704#issuecomment-1850711056